packages/core/src/telemetry/service.ts

```diff
@@ -134,23 +133,24 @@ export class TelemetryService {
   }
 
   private flushDetached() {
-    if (this.events.length === 0) {
-      return;
-    }
+    if (this.events.length === 0) return;
 
     const serializedEvents = JSON.stringify(this.events);
-    const fileName = path.join(this.options.ponderDir, "telemetry-events.json");
-    fs.writeFileSync(fileName, serializedEvents);
+    const telemetryEventsFilePath = path.join(
+      this.options.ponderDir,
+      "telemetry-events.json"
+    );
+    fs.writeFileSync(telemetryEventsFilePath, serializedEvents);
+
     child_process.spawn(process.execPath, [
       path.join(__dirname, "flush-detached.js"),
-      fileName,
+      this.options.telemetryUrl,
```

Comment:
Made the telemetry service URL  an argument to the script and passed it along here. Makes easier unit testing and allows us to only store the telemetry URL in one place.